### ISSUE TITLE
Zemberek component cannot be disabled in enchant 1.6.1

### ISSUE THREAD

**Issue by shibotto** [rrthomas/enchant#199]
The quicker way to reproduce this for me is to launch `/usr/bin/gspell-app1`:

```
shiba ~ > gspell-app1
Available language codes: en_AU, en_CA, en_GB, en_US, en_ZA, it_IT
fish: 'gspell-app1' terminated by signal SIGSEGV (Address boundary error)
```

Upgrading enchant to 2.1.2 or downgrading to 1.6.0 solves the problem. Here's my distro build instruction if you need it https://gitweb.gentoo.org/repo/gentoo.git/tree/app-text/enchant/enchant-1.6.1.ebuild

Can you reproduce this?

**Comment by rrthomas** [comment]
I'm not fixing bugs in 1.6.x, but if there's demand I'm happy to accept patches.

**Comment by shibotto** [comment]
Thank you for your answer. I did a bit more testing and I found out the issue is the Zemberek component. I'm not trying to fix the component though, but as of 1.6.1 I cannot even disable it anymore, so I'll change the bug to that.

It all began with bb2850c172459177a0d03304f16166bc6013e49e , which fixed dbus-glib detection. However I believe there is something wrong in the `configure.ac`, because having dbus-glib installed enables Zemberek despite explicitly specifying `--disable-zemberek` during config.

Sadly I have no experience with autotools so I don't know how to properly fix this, but deleting https://github.com/AbiWord/enchant/blob/enchant-1-6-1/configure.ac#L513 makes `--disable/enable-zemberek` work again. I'll look for someone to decently patch this.

**Comment by puleglot** [comment]
Gentoo PR with a patch for configure.ac: https://github.com/gentoo/gentoo/pull/9392

**Comment by rrthomas** [comment]
I should emphasise, it should not be necessary to use 1.6.1. If there's any problem with programs that won't build with the current version, I'm happy to help their upstream fix that.

**Comment by shibotto** [comment]
This was fixed for 1.6.1 in Gentoo, which in the meantime is transitioning to >=2.x. Thank you @puleglot for reporting this to Gentoo and thank you @rrthomas for offering support, though I wasn't of much help XD
